man/os-release.xml

```diff
@@ -309,6 +309,22 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CODENAME=</varname></term>
+
+        <listitem><para>
+        A lower-case string (no spaces or other characters outside of
+        0–9, a–z, ".", "_" and "-") identifying the operating system
+        release code name, excluding any OS name information or
+        release version, and suitable for processing by scripts or
+        usage in generated filenames. This field is optional and may
+        not be implemented on all systems.
```

Comment:
I suppose that's up for bikeshedding; `VERSION_CODENAME=` makes some sense as the other fields like `ID=` or `SUPPORT_URL=` are not specific to the OS version. But then again, `PRETTY_NAME=` *is* specific to the OS version, so it wouldn't be consistent either way.

Still, I agree that `VERSION_CODENAME=` is a bit more consistent indeed.